src/arch/riscv/early_mem.c

```diff
@@ -43,37 +43,38 @@ extern ulong_t kernel_end;
 off_t dtb_ram_start = 0;
 size_t dtb_ram_size = 0;
 
-bool_t dtb_node_get_rsv_ram (struct dtb_node *n) {
-    if(strstr(n->name, "mmode_resv")) {
-        addr_t start = n->reg.address;
-        ulong_t len = n->reg.length;
-        INFO_PRINT("Reseving %s region (%p, size %lu)\n", n->name, start, len);
-        mm_boot_reserve_mem(start, len);
-    }
-    return true;
-}
-
-bool_t dtb_node_get_ram (struct dtb_node * n) {
-    if(!strcmp(n->name, "memory")) {
-        dtb_ram_size = n->reg.length;
-        dtb_ram_start = n->reg.address;
-        return false;
-    }
-    return true;
-}
+// bool_t dtb_node_get_rsv_ram (struct dtb_node *n) {
```

Comment:
Let's just delete this commented code if the rest works :)

Reply:
Yup, ended up just reimplementing the commented part because it was pretty simple